<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfigurationPage</class>
 <widget class="QWidget" name="GeneralConfigurationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>846</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_14">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="title">
      <string>User interface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="1" column="0">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="uiLanguage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Use system language setting</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="isHighDPIScalingEnabled">
        <property name="text">
         <string>Enable high DPI scaling</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="applicationName">
        <property name="text">
         <string>iTALC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>iTALC Service</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0" colspan="5">
       <widget class="QCheckBox" name="isTrayIconHidden">
        <property name="text">
         <string>Hide tray icon</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QCheckBox" name="autostartService">
        <property name="text">
         <string>Autostart</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>24</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Additional arguments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serviceArguments"/>
        </item>
       </layout>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="startService">
        <property name="text">
         <string>Start service</string>
        </property>
        <property name="icon">
         <iconset resource="../italc-configurator.qrc">
          <normaloff>:/resources/media-playback-start.png</normaloff>:/resources/media-playback-start.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="stopService">
        <property name="text">
         <string>Stop service</string>
        </property>
        <property name="icon">
         <iconset resource="../italc-configurator.qrc">
          <normaloff>:/resources/media-playback-stop.png</normaloff>:/resources/media-playback-stop.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="serviceState">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Stopped</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QCheckBox" name="lockWithDesktopSwitching">
        <property name="text">
         <string>Enable desktop switching for screen lock (experimental)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Logging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Log file directory</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="logFileDirectory"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="openLogFileDirectory">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../italc-configurator.qrc">
            <normaloff>:/resources/document-open.png</normaloff>:/resources/document-open.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Log level</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="logLevel">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Nothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Only critical messages</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Errors and critical messages</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warnings and errors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Information, warnings and errors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Debug messages and everything else</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="limittedLogFileSize">
          <property name="text">
           <string>Limit log file size</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="logFileSizeLimit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="logToStdErr">
        <property name="text">
         <string>Log to standard error output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="logToWindowsEventLog">
        <property name="text">
         <string>Log to Windows event log</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>281</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clearLogFiles">
          <property name="text">
           <string>Clear all log files</string>
          </property>
          <property name="icon">
           <iconset resource="../italc-configurator.qrc">
            <normaloff>:/resources/edit-delete.png</normaloff>:/resources/edit-delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../italc-configurator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>limittedLogFileSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>logFileSizeLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutItalc()</slot>
 </slots>
</ui>
